include/Tensor2_3D.hpp

```diff
@@ -2,7 +2,7 @@
 #define TENSOR2_3D_HPP
 // ============================================================================
 // Tensor2_3D.hpp
-// This is a 2th-order tensor in 3D. The components are stored in a 1-D array: 
```

Comment:
Good! 👍

Reply:
@Goldeninscriptions I have merged this with master. Please continue your review in master branch.

Reply:
OK